recipes/snntorch/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "snntorch" %}
+{% set version = "0.4.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snntorch-{{ version }}.tar.gz
+  sha256: 44223a35cf8f131ed0dd41af997735f4cd97fc1b30d83f9b3ad28b48ac9d697f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch-cpu
+  run:
+    - python >=3.6
+    - pytorch-cpu
+    - numpy
+    - pandas
+    - matplotlib
```

Unless you need a pyqt-based GUI, better to depend on matplotlib-base:
```suggestion
    - matplotlib-base
```